src/plugins/query_enhancements/public/plugin.tsx

```diff
@@ -100,6 +100,7 @@ export class QueryEnhancementsPlugin
       editor: enhancedSQLQueryEditor,
       editorSupportedAppNames: ['discover'],
       supportedAppNames: ['discover', 'data-explorer'],
+      disableDatePicker: true,
```

Comment:
nit: this my fault for suggesting something inaccurate and feel free to ignore this comment. but after thinking about it `disable` for me would make me believe that this will still render it but show it disabled. If it does that then this is perfect name but if it's hiding then perhaps we should go with the `showDatePicker`

Reply:
+1